processor/src/main/kotlin/com/tobrun/datacompat/DataCompatProcessor.kt

```diff
@@ -327,46 +363,36 @@ class DataCompatProcessor(
                         )
                         .addKdoc(
                             """
-                            |Set $kDocProperty
-                            |${property.key.docString?.trimStart(' ', '\n') ?: ""}
-                            |@param $propertyName $kDocProperty
+                            |Setter for ${property.value.kDoc.trimEnd('.')
```

Comment:
Nit: This doesn't always work. For example:
```
* Setter for defines text color of the attribution icon.
```

Would it be too redundant to do:
```suggestion
                            |Setter for $propertyName: ${property.value.kDoc.trimEnd('.')
```

Reply:
Yeah, that slipped away from me 😄 will fix